### PR TITLE
Added Invoke method -> call java code to set wallpaper

### PR DESCRIPTION
**Pull request by 1-2-ka-4-4-2-ka-1**
Added Invoke method -> call java code to set wallpaper
+Added Glide dependency to android code.

### CODE REVIEW

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```
Comment (codenameakshay) [inline android/gradle/wrapper/gradle-wrapper.properties]: Isme change kyu kiya?

File: android/build.gradle [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-    ext.kotlin_version = '1.3.50'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.3.3'
```
Comment (codenameakshay) [inline android/build.gradle:11]: Yeh kyu hataya com.google.gms? Yeh Firebase ke liye chahiye hota hai

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (UnHired-Coder) [reply android/gradle/wrapper/gradle-wrapper.properties]: Mera gradle version dusra hai.......fixed it.

File: android/build.gradle (hunk 2 above)
Comment (UnHired-Coder) [reply android/build.gradle:11]: See, default code was kotlin+dart, so I had to delete the android directory and create a new java directory, please add the necessary files again.
services JSON, and if any permissions you requested to the android manifest.

**Pull request merged**